### PR TITLE
Add FTGL configure flag

### PR DESCRIPTION
**Pull request by deltaoscarmike**
Not enabled by default.

### CODE REVIEW

**Comment by revmischa** [comment]
libftgl needed for OSX travis build https://travis-ci.org/projectM-visualizer/projectm/jobs/362786185

**Comment by revmischa** [comment]
shipit!

File: src/libprojectM/Common.hpp [hunk 1]
```diff
@@ -30,6 +30,7 @@
 #include <typeinfo>
 #include <cstdarg>
 #include <cassert>
+#include "config.h"
```
Comment (revmischa) [inline src/libprojectM/Common.hpp]: in re https://github.com/projectM-visualizer/projectm/pull/53#discussion_r180357094 -
is this necessary in Common.hpp? this header is (probably) needed to be "public" and shouldn't include config.h which is "private" to the project

Comment (deltaoscarmike) [reply src/libprojectM/Common.hpp]: It could be deleted, I should have defined "USE_FTGL" with a cflag instead.

**Pull request merged**